Cargo.toml

```diff
@@ -20,10 +20,3 @@ resolver = "2"
 
 [workspace.metadata.cargo-machete]
 ignored = ["prost", "workspace-hack"]
-
-[patch.crates-io]
-# This branch update the tonic version for madsim. We should switch to the original etcd-client crate when new version release.
-madsim = { git = "https://github.com/Phoenix500526/madsim.git", branch = "update-tonic" }
-madsim-tonic = { git = "https://github.com/Phoenix500526/madsim.git", branch = "update-tonic" }
-madsim-tonic-build = { git = "https://github.com/Phoenix500526/madsim.git", branch = "update-tonic" }
-madsim-tokio = { git = "https://github.com/Phoenix500526/madsim.git", branch = "update-tonic" }
```

Comment:
Should we maintain a fork that tracks the upstream? It seems we constantly have to wait for a release in madsim.

Reply:
OK.

Reply:
I think we should maintain the fork under xline-kv organization rather than personal account. We could address this in another PR instead of this one.